kin-sdk/src/androidTest/java/kin/sdk/KinAccountTest.java

```diff
@@ -16,6 +16,7 @@
 public class KinAccountTest {
```

Comment:
should add tests here for other new operations on `KinAccount`, to make sure they throw `AccountDeletedException` when needed

Reply:
the new operation 'sendWhitelistTransaction' is sending the transaction just like the other regular sendTransaction, the only change is the parameter but it is using the same code so i don't think there is a need for the same test.